logic/raid_list.php

```diff
@@ -11,6 +11,10 @@ function raid_list($update)
     // Init raid id.
     $iqq = 0;
 
+    // Need db functions real_escape_string and use config
+    global $db;
```

Comment:
`$db` was from mysqli, it has since been deprecated so `$dbh` & PDO is the way to go instead.